indico/modules/events/surveys/templates/manage_survey_state/finished.html

```diff
@@ -14,7 +14,8 @@
 {% block now_button %}
     <div class="group">
         <a href="#" data-href="{{ url_for('.open_survey', survey) }}" data-method="POST"
-           data-confirm="{% trans %}The survey finished already. Do you want to open it again for submissions?{% endtrans %}"
+           data-confirm="{% trans %}The survey finished already. Do you want to open it again for submissions?
+           This will result in resending the survey start notification.{% endtrans %}"
```

Comment:
i'd use different messages depending on whether the start notification is actually enabled or not. or at least add "if it is enabled"